### DISCUSSION TITLE
Preserve fingerprint when moving mosquito to another server

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#22015] [Q&A]
When I move mosquito to another server and get a certificate from letsencrypt for the same domain the connection always fails because the fingerprint differs. How can I prevent this?

**Comment by s-hadinger** [comment]
The fingerprint is derived from the public key of the server, which is derived from the private key. If both servers have the same private keys, they should have the same fingerprints.
I'm not sure if letsencrypt allows to share private keys between servers.
